src/services/Subscriptions.php

```diff
@@ -280,4 +284,39 @@ public function getUnsavedDraftByUid(StripeSubscription $subscription): Subscrip
             ->drafts()
             ->one() ?? new SubscriptionElement();
     }
+
+    /**
+     * Ensures that a user with given email address is created if one doesn't already exist.
+     *
+     * @param StripeSubscription $subscription
+     * @param SubscriptionElement $subscriptionElement
+     * @return void
+     * @throws \yii\base\Exception
+     * @throws \yii\base\InvalidConfigException
+     */
+    private function maybeEnsureUser(StripeSubscription $subscription, SubscriptionElement $subscriptionElement): void
```

Comment:
I would be tempted to have this as just `ensureUser()` and then wrap the call to it in a conditional. 

Can make things a little clearer in the fact that the method has "one job".

Not a biggie though so will let you decide on this one

Reply:
That makes sense. Updated.